File: FizzBuzz/fb.rb
# fizz buzz solution in ruby for code eval by steven a dunn

File.open(ARGV[0]).each_line do |line|
  a, b, n = line.split.map! {|x| x = x.to_i }
  (1..n).each do |i|
    result = ""
    if i % a == 0
      result += "F"
    end
    if i % b == 0
      result += "B"
    end
    if result.empty?
      print "#{i} "
    else
      print "#{result} "
    end
  end
  puts
end

File: FizzBuzz/JS/fb.js
// Fizz Buzz solution in Node for CodeEval.com by Steven A. Dunn

var fs = require("fs");
fs.readFileSync(process.argv[2]).toString().split('\n').forEach(function (line) {
  if (line !== "") {

    var split = line.split(" ");
    var A = split[0];
    var B = split[1];
    var N = split[2];

    var output = "";
    for (var i = 1; i < parseInt(N) + 1; ++i) 
    {
      var aMod = i % parseInt(A);
      var bMod = i % parseInt(B);

      if (aMod === 0 && bMod === 0)
        output += "FB";
      else if (aMod === 0)
        output += "F";
      else if (bMod === 0)
        output += "B";
      else
        output += i;
      output += " ";
    }
    console.log(output);
  }
});
